frontend/scripts/flutter_release_build/build_flowy.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+
+part 'tool.dart';
+
+Future<void> main(List<String> args) async {
+  const help = '''
+A build script that modifies build assets before building the release version of AppFlowy.
+
+args[0]: The directory that contains the AppFlowy git repository. Should be the parent to appflowy_flutter. (absolute path)
+''';
+  const numArgs = 1;
+  assert(args.length == 1,
```

Comment:
```suggestion
  const numArgs = 1;
  assert(args.length == numArgs,
```

Reply:
This was fixed in https://github.com/AppFlowy-IO/AppFlowy/pull/2106 and will be on main once it lands.